### PR TITLE
Feature/documentation version dropdown

### PR DESCRIPTION
**Pull request by hiltpold**
### What changes are included in the pull request?
A dropdown has been added to the JsonSchemaViewer from the documentation. It is now possible to change the version of the schema and display it. The current exporter of the schema has to be updated, as the "id" field in the schema json should equal the filename.

### Why are the changes needed?
Better usability of the documentation.

### CODE REVIEW

**Review by Geheiner — APPROVED** [review]
LGTM, seems to work, tried it in my browser. FYI I changed the base branch to documentation.

**Comment by mand35** [comment]
This is a neat feature. 
I am not sure if this is necessary, but it would be more consistent if the link to share also reference to the version selected. Otherwise we may reference to an object which does not exists anymore in the latest version.

File: static/json-schema-viewer/index.html [hunk 1]
```diff
@@ -6,7 +6,17 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-mobile/1.4.5/jquery.mobile.min.css" />
         <link rel="stylesheet" href="styles/json-schema-viewer.min.css" />
-
+        <style>
+            .dropdown-container{
+                display: flex;
+                align-items: center;
+                display: flex;
```
Comment (mand35) [inline static/json-schema-viewer/index.html:13]: Is that repetition of `display` and `align-items` necessary here?

Comment (hiltpold) [reply static/json-schema-viewer/index.html:13]: Good catch. Might be artifacts from me using vim :)

**Comment by zzeekk** [comment]
> This is a neat feature. I am not sure if this is necessary, but it would be more consistent if the link to share also reference to the version selected. Otherwise we may reference to an object which does not exists anymore in the latest version.

Good point. It would be nice to have to add the version as URL parameter...

**Comment by mand35** [comment]
looks great with that fixed link now. 👍

**Review by zzeekk — APPROVED** [review]

**Pull request merged**